project/Dependencies.scala

```diff
@@ -7,9 +7,9 @@ object Dependencies {
   lazy val circeParser = "io.circe" %% "circe-parser" % "0.13.0"
   lazy val circeGeneric = "io.circe" %% "circe-generic" % "0.13.0"
   lazy val circeGenericExtras = "io.circe" %% "circe-generic-extras" % "0.13.0"
-  lazy val sttp = "com.softwaremill.sttp.client" %% "core" % "2.0.0-RC9"
-  lazy val sttpCirce = "com.softwaremill.sttp.client" %% "circe" % "2.0.0-RC9"
-  lazy val sttpAsyncClient = "com.softwaremill.sttp.client" %% "async-http-client-backend-future" % "2.0.0-RC9"
+  lazy val sttp = "com.softwaremill.sttp.client" %% "core" % "2.2.4"
```

Comment:
Is it worth abstracting the sttp version to a variable as its repeated multiple times, ot make it easier to update in the future?

Reply:
Yep, I've done that. I've also commonised the Circe version.